core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -203,6 +204,7 @@ contract Liquidatable is PricelessPositionManager {
                 settlementPrice: FixedPoint.fromUnscaledUint(0)
             })
         );
+        uint newLength = liquidations[sponsor].length;
```

Comment:
nit: since we're only using this to get the uuid (length - 1), you could do something like this instead:
```
uuid = liquidations[sponsor].length;
liquidations[sponsor].push( ... );
```

Reply:
done